frontends/p4/simplifyDefUse.cpp

```diff
@@ -132,17 +180,19 @@ class FindUninitialized : public Inspector {
 
     Definitions* getCurrentDefinitions() const {
         auto defs = definitions->getDefinitions(currentPoint, true);
-        LOG3("FU Current point is (after) " << currentPoint << " definitions are " << defs);
+        LOG3("FU Current point is (after) " << currentPoint <<
```

Comment:
What does the 'FU' acronym stands for? =)

Reply:
Presumably "Find Uninitialized." But it could easily be confused for something else.